kubewarden/kwctl
# Save and load scripts

These scripts are required for airgap instalation. 

- `kubewarden-save-policies.sh` saves all policies in the file provided inside a tar.gz
- `kubewarden-load-policies.sh` load policies inside a tar.gz 

Steps to test it:

policies.txt example
```
registry://ghcr.io/kubewarden/policies/capabilities-psp:v0.1.9
registry://ghcr.io/kubewarden/policies/psp-capabilities:v0.1.5
https://github.com/kubewarden/pod-privileged-policy/releases/download/v0.1.6/policy.wasm

```

- Download kwctl 1.3.2
- Create a private registry
-  .`/kubewarden-save-policies.sh --policies-list policies.txt --policies policies.tar.gz`
- remove all policies in `policies.txt` with `kwctl rm`
- `./kubewarden-load-policies.sh --policies policies.tar.gz --policies-list policies.txt --registry localhost:5000 --sources-path sources.yml`
- verify that the following policies are present in the private registry:
```
localhost:5000/kubewarden/policies/capabilities-psp
localhost:5000/kubewarden/policies/psp-capabilities
localhost:5000/kubewarden/pod-privileged-policy/releases/download/v0.1.6/policy.wasm
```

Fix https://github.com/kubewarden/kubewarden-controller/issues/344
